earthpy/__init__.py

```diff
@@ -6,7 +6,7 @@
 import os.path as op
 from . import utils, spatial
 
-data = EarthlabData()
+earthlabdata = EarthlabData()
```

Comment:
@mbjoseph i like .data as it keeps the call simple. May i ask why you renamed it earthlab data? the idea with this is it doesn't have to be earthlab data. it can be any data. we've updated the class to accept url's now too to facilitate using the hub. so i'd rather not make this change.

Reply:
Here is a page where we call to get the data
https://www.earthdatascience.org/courses/earth-analytics-python/multispectral-remote-sensing-in-python/ but we use it on every lesson everywhere!!

you can also call this:

`et.data.get_data(url="https://ndownloader.figshare.com/files/12459464")` which again allows you to get ANY data and unpack it in the same way that you can unpack our prepackaged data. My goal for this function is it's a way for a class  using earthpy to make it easier for students to get data into an established working dir. 

We could consider renaming the `earthpy/data` directory rather than the instance defined in the init file in this case.

Reply:
Cool thank you @lwasser! I suspected it was being used already. I agree renaming the current `earthpy/data/` directory seems like a good solution (and a way to avoid confusion for others, who would expect `earthpy.data` to relate to the `earthpy/data/` directory).

Reply:
Reverted back to `earthlab.data` in b85c98482b504f3704d3693fc72bafc774b7c952